src/MongoDB.Driver.Core/Core/Operations/FindCommandOperation.cs

```diff
@@ -315,6 +315,7 @@ public BsonDocument Min
         /// <value>
         ///   <c>true</c> if the OplogReplay bit will be set; otherwise, <c>false</c>.
         /// </value>
+        [Obsolete("OplogReplay is ignored by server versions 4.3.1 and newer.")]
```

Comment:
as it was discussed in the scope of CSHARP-2738, let's use 4.4.0 version for obsolete messages. Similar in other places.

Reply:
Done